modules/aleph/src/Aleph/AlephClient.php

```diff
@@ -175,9 +182,39 @@ public function changePin(AlephPatron $patron, $new_pin) {
    *
    * @return \SimpleXMLElement
    *    The SimpleXMLElement response from Aleph.
+   *
+   * @throws \RuntimeException
    */
   public function getDebts(AlephPatron $patron) {
-    return $this->requestRest('GET', 'patron/' . $patron->getId() . '/circulationActions/cash?view=full', array());
+    return $this->requestRest('GET', 'patron/' . $patron->getId() . '/circulationActions/cash?view=full');
+  }
+
+  /**
+   * @param \Drupal\aleph\Aleph\AlephMaterial $material
+   *    The Aleph material to get items from.
+   *
+   * @return \SimpleXMLElement
+   *    The SimpleXMLElement response from Aleph.
+   *
+   * @throws \RuntimeException
+   */
+  public function getItems(AlephMaterial $material) {
+    return $this->requestRest('GET', 'record/' . 'ICE01' . $material->getId() . '/items?view=full');
```

Comment:
"ICE01" should at the very least be a constant, preferable a variable so that this becomes configurable.